lib/mfu/mfu.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file mfu.h
+ *
+ * @defgroup mfu Modem firmware update library
+ * @{
+ * @brief Library for verifying and installing a downloaded modem firmware image.
+ *
+ * @details This library contains functionality to reads, validates, and install
+ * a modem update image that has been downloaded to a flash device.
+ * A modem image is generally too large to fit in nRF9160 internal flash,
+ * which necessitates the use of external non-volatile memory.
+ *
+ * Verification is done via SHA256 hash, and optionally an ECDSA signature.
+ *
+ * Persistent state information is kept in order to resume an interrupted
+ * update.
+ *
+ */
+
+#ifndef __MFU_H__
+#define __MFU_H__
+
+#include <zephyr/types.h>
+#include <device.h>
```

Comment:
Maybe it's just me but I can't see where this is used in the header file.

Reply:
Oh yeah, that's from a previous iteration of mfu_init, where there was a `struct device *flash_device` parameter instead of a device name string. Should be moved to mfu.c